src/components/common/ErrorsDisplay.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { array } from 'prop-types';
+
+const DisplayErrors = ({ errors }) => {
+  if (!errors) {
+    return;
+  }
+  return (
+    <ul className="error-bullets">
+      {errors.map((item, index) => <li key={index}>{item}</li>)}
```

Comment:
Errors is not always an array. You should account for that.

Reply:
Fixed!